docs/Project-Configuration/Project-Commands.md

```diff
@@ -533,6 +533,7 @@ Parameters:
     means that for patch builds, editing the
     ["modules"](Project-Configuration-Files#modules)
     section of the project config will not change the checked out hash.
+    If you do not specify any revisions, all of them will be cloned.
```

Comment:
GH doesn't let me comment on unchanged lines but while we're at it, can we remove the list `-` from L516? Modules is a top-level map key, not a list element.